yt/visualization/base_plot_types.py

```diff
@@ -6,36 +6,35 @@
 
 from yt.funcs import (
     get_brewer_cmap,
-    get_image_suffix,
     get_interactivity,
     iterable,
     matplotlib_style_context,
     mylog,
 )
 
-backend_dict = {
+from ._commons import get_canvas, validate_image_name
+
+BACKEND_SPECS = {
```

Comment:
nothing is currently achieved by this change. I was trying to look if this dictionary could be useful elsewhere, and refactored a bit around it to make sense out of it. Now I don't think this would actually be useful as part of yt/visualization/_commons.py but the code that uses it is now slightly more readable, so I think I'll just keep it as is.